### PR TITLE
Add missing unit tests

### PR DESCRIPTION
**Pull request by rowanmanning**

### CODE REVIEW

File: test/unit/lib/output.test.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		});
+
+		it('should have a `stream` property containing a stream representation of `bufferedContent`', function() {
+			assert.notCalled(streamCat);
+			assert.strictEqual(instance.stream, streamCat.firstCall.returnValue);
+			assert.calledOnce(streamCat);
+			assert.deepEqual(streamCat.firstCall.args[0], [bufferedContent]);
+		});
+
+		it('should have a `pipe` method', function() {
```
Comment (AlbertoElias) [inline test/unit/lib/output.test.js]: Is this one and the following necessary? They seem to be more streamcat/node.js tests

Comment (rowanmanning) [inline test/unit/lib/output.test.js]: I'd agree if we were extending Stream or something, but the Output object implements the pipe method itself https://github.com/Financial-Times/origami-build-service/blob/master/lib/output.js#L25

File: test/unit/lib/utils/uniqueid.test.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+		describe('when `id.length` is greater than `maxLength`', function() {
+			let id;
+
+			beforeEach(function() {
+				crypto.createHash.returns({
+					digest: sinon.stub().returns('foobarbazqux1234567890abcdef'),
+					update: sinon.stub()
+				});
+				id = 'hello-world-how-are-you-today-foo-bar-baz';
```
Comment (AlbertoElias) [inline test/unit/lib/utils/uniqueid.test.js]: I'm doing good thanks, I've got Jupiter behind me so I'm trying to cast a shadow on it, wish me luck!

**Comment by AlbertoElias** [comment]
Looks good!

**Pull request merged**